vulnerabilities/views.py

```diff
@@ -102,8 +110,12 @@ def request_to_vulnerabilities(request):
         vuln_id = request.GET["vuln_id"]
         return list(
             models.Vulnerability.objects.filter(vulnerability_id__icontains=vuln_id).annotate(
-                vulnerable_package_count=Count("vulnerable_packages"),
-                patched_package_count=Count("patched_packages"),
+                vulnerable_package_count=Count(
+                    "packages", filter=Q(packagerelatedvulnerability__fix=False)
```

Comment:
The query for affected and fixed packages should be abstracted as methods on the models (and/or on a model manager)